kokoro/gcp_ubuntu/kokoro_build.sh

```diff
@@ -8,9 +8,9 @@ set -x
 
 #Variables
 GCP_PROJECT=$(gcloud config get-value project)
-GCP_PROJECT_ID=$(gcloud projects describe ${GCP_PROJECT} --format 'value(projectNumber)')
-GCP_SERVICE_ACCOUNT=$(${GCP_PROJECT_ID}-compute@developer.gserviceaccount.com)
-GCP_IMAGE=$(projects/${GCP_PROJECT}/global/images/teradata1610-ubuntu20)
+GCP_PROJECT_ID=$(gcloud projects describe "${GCP_PROJECT}" --format 'value(projectNumber)')
+GCP_SERVICE_ACCOUNT='"${GCP_PROJECT_ID}"-compute@developer.gserviceaccount.com'
+GCP_IMAGE='projects/"${GCP_PROJECT}"/global/images/teradata1610-ubuntu20'
```

Comment:
Do you really want single quotes here? That will cause the entire string to be seen as a literal and the inner `${...}` variable will not be expanded.